sseft_shap_new.py

```diff
@@ -0,0 +1,189 @@
+
+
+import shap
+import torch
+import numpy as np
+from PIL import Image
+import os, copy, sys
+import math, json
+import random
+from tqdm import tqdm
+from fairseq import checkpoint_utils, options, progress_bar, utils
```

Comment:
Many dependencies have not been declared in pyproject.toml yet
Also, quite some imports are never used in the module
And the submodules (fairseq) have their own requiremens that are not listed in pyproject.toml